content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Upgrade Notes
+description: Important Changes to consider when upgrading to Istio 1.6.
+weight: 20
+---
+
+This page describes changes you need to be aware of when upgrading from Istio
+1.5.x to Istio 1.6.x. Here, we detail cases where we intentionally broke backwards
+compatibility. We also mention cases where backwards compatibility was preserved
+but new behavior was introduced that would be surprising to someone familiar with
+the use and operation of Istio 1.5.
+
+{{< tip >}}
+Istio does not currently support skip-level upgrades. For example, if you are still using Istio 1.4, we recommend first upgrading to Istio 1.5. However, if you choose to upgrade from previous version, you must first disable Galley configuration validation. This can be done by adding `--enable-validation=false` to the Galley deployment and removing the `istio-galley` `ValidatingWebhookConfiguration`
+{{< /tip >}}
+
+## Removal of legacy Helm charts
+
+In Istio 1.4 we introduced a [new way to install Istio](/blog/2019/introducing-istio-operator/), using the in-cluster Operator or `istioctl install` command. As part of this effort, we deprecated the old Helm charts in 1.5. Over time, we implemented many of the new Istio features only in these new installation methods. As a result, we have decided to remove the old installation Helm charts in Istio 1.6.
+
+We recommend reviewing the [Istio 1.5 Upgrade Notes](/news/releases/1.5.x/announcing-1.5/upgrade-notes/#control-plane-restructuring) before continuing, because we introduced several changes in Istio 1.5 that were not present in the legacy installation method, such as Istiod and Telemetry V2.
+
+You can now safely upgrade from the legacy Helm charts using a [Control Plane Revision](/blog/2020/multiple-control-planes/). In place upgrade is not supported and may result in downtime, so please follow the [Canary Upgrade](/docs/setup/upgrade/#canary-upgrades) steps.
+
+## v1alpha1 security policy is not supported anymore
+
+Istio 1.6 no longer supports the [`v1alpha1` authentication policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.authentication.v1alpha1/) and [`v1alpha1` RBAC policy](https://archive.istio.io/v1.4/docs/reference/config/security/istio.rbac.v1alpha1/), these `v1alpha1` APIs will be ignored starting 1.6.
```

Comment:
@rcaballeromx 
How about this?
Starting in Istio 1.6, these `v1alpha1` security policy APIs are ignored.

Instead of: 
Istio 1.6 starts ignoring these `v1alpha1` security policy APIs.